Directory.Build.props

```diff
@@ -1,15 +1,15 @@
 <Project>
   <PropertyGroup>
     <VersionPrefix>3.0.0</VersionPrefix>
-    <AssemblyCopyright>Copyright (c) 2006-2020 The Contributors of the Python.NET Project</AssemblyCopyright>
+    <AssemblyCopyright>Copyright (c) 2006-2022 The Contributors of the Python.NET Project</AssemblyCopyright>
```

Comment:
Bold move ;)

Reply:
No worries, I committed this using my DeLorean. :-D

Reply:
Just fix this one, please and you can go ahead and merge :)